jekyll/_includes/snippets/pipeline-values.adoc

```diff
@@ -50,6 +50,13 @@
 | icon:check[]
 | icon:check[]
 
+| `pipeline.git.branch.is_default`
+| GitHub App, GitHub OAuth, Bitbucket, GitLab
+| Boolean
+| Whether the branch the pipeline was triggered on is the default branch.
+| icon:check[]
+| icon:check[]
```

Comment:
When does this become available to server customers? which version?

Reply:
4.6 and above for Server customers and all pipelines created after 31st July 2024 for cloud customers.

There's no current plans to backfill existing pipelines in either environment.